content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+sedikit seiring berevolusinya perangkat, namun secara umum implementasinya sudah cukup stabil.
+
+{{< note >}}
+Semua perintah di dalam `kubeadm alpha`, sesuai definisi, didukung dalam level alfa.
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Tujuan
+
+* Memasang Kubernetes klaster dengan _control-plane_ tunggal atau [klaster dengan ketersediaan tinggi](/docs/setup/production-environment/tools/kubeadm/high-availability/)
+* Memasang jaringan Pod pada klaster sehingga Pod dapat
+  berinteraksi satu sama lain
+
+## Instruksi
+
+### Memasang kubeadm pada _host_
+
+Lihat ["Memasang kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
+
+{{< note >}}
+Jika kamu sudah memasang kubeadm sebelumnya, jalankan `apt-get update &&
+apt-get upgrade` atau `yum update` untuk mendapatkan versi kubeadm paling baru.
+
+Ketika kamu melakukan pembaruan, kubelet melakukan _restart_ setiap beberapa detik sambil menunggu dalam _crashloop_ sampai
+kubeadm memberikan perintah yang harus dilakukan. _Crashloop_ ini memang diantisipasi dan normal.
+Setelah kamu menginisialisasi _control-plane_, kubelet akan berjalan normal.
+{{< /note >}}
+
+### Menginisialisasi simpul _control-plane_
+
+Simpul _control-plane_ adalah mesin dimana komponen-komponen _control plane_ berjalan, termasuk
+{{< glossary_tooltip term_id="etcd" >}} (basis data klaster) dan
+{{< glossary_tooltip text="peladen API" term_id="kube-apiserver" >}}
+(yang akan berkomunikasi dengan perangkat _command line_ {{< glossary_tooltip text="kubectl" term_id="kubectl" >}}.
+
+1. (Direkomendasikan) Jika kamu berencana untuk memperbarui klaster `kubeadm` _control-plane_ tunggal 
+menjadi ketersediaan tinggi kamu harus menentukan `--control-plane-endpoint` agar mengarah ke _endpoint_ yang digunakan bersama
+untuk semua simpul _control-plane_. _Endpoint_ tersebut dapat berupa nama DNS atau sebuah alamat IP dari _load-balancer_.
+1. Pilih _add-on_ jaringan Pod, dan pastikan apakah diperlukan argumen untuk
+diberikan pada `kubeadm init`. Tergantung
+penyedia pihak ketiga yang kamu pilih, kamu mungkin harus mengatur `--pod-network-cidr` dengan nilai
+yang spesifik pada penyedia tertentu. Lihat [Memasang _add-on_ jaringan Pod](#jaringan-pod).
+1. (Opsional) Sejak versi 1.14, `kubeadm` mencoba untuk mendeteksi _runtime_ kontainer pada Linux
+dengan menggunakan daftar _domain socket path_ yang umum diketahui. Untuk menggunakan _runtime_ kontainer yang berbeda atau
+jika ada lebih dari satu yang terpasang pada simpul yang digunakan, tentukan argumen `--cri-socket`
+pada `kubeadm init`. Lihat [Memasang _runtime_](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime).
+1. (Opsional) Kecuali ditentukan sebelumnya, `kubeadm` akan menggunakan antarmuka jaringan yang diasosiasikan
+dengan _default gateway_ untuk mengatur alamat _advertise_ untuk peladen API pada simpul _control-plane_ ini.
```

Comment:
```suggestion
dengan _default gateway_ untuk mengatur alamat _advertise_ untuk API Server pada Node _control-plane_ ini.
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc